paseos/actors/actor_builder.py

```diff
@@ -305,6 +306,63 @@ def set_position(actor: BaseActor, position: list):
         actor._position = position
         logger.debug(f"Setting position {position} on actor {actor}")
 
+    @staticmethod
+    def set_cuboid_geometric_model(
+            actor: SpacecraftActor,
+            mass: float,
+            height: float,
+            length: float,
+            width: float,
+    ):
+        """Define geometry of the spacecraft actor.
+
+        Args:
+            actor (SpacecraftActor): Actor to update.
+            mass (float): Mass of the spacecraft in kg
+            height (float): Size of the spacecraft in z (nadir) direction in meters
+            length (float): Size of the spacecraft in y (along-track) direction in meters
+            width (float): Size of the spacecraft in x (across-track) direction in meters
+        """
+        assert mass >= 0, "Mass is <= 0"
```

Comment:
```suggestion
        assert mass > 0, "Mass is > 0"
```
zero mass is probably gonna blow up some point, would stick to > 0